### ISSUE TITLE
Where are the transcription results?

### ISSUE THREAD

**Issue by skyInGitHub** [mravanelli/pytorch-kaldi#126]
Hi, I built up a model successfully with training, dev and test sets. And get 3 folders including generated_outputs, exp_files, and decode_test_out_dnn1. 

Where can I find the final output (transcriptions) of the audio in the test set transcribed by the built model?

**Comment by TParcollet** [comment]
Hi !

If everything went fine during decoding time. you may find a .log in decode_test_out_dnn1/log . If you want a clean transcription, you might need to check the lattice-align / lattice-push / lattice-to-ctm commands.
